src/Bindings/ManualBindings.cpp

```diff
@@ -2975,8 +2975,8 @@ static int tolua_cLineBlockTracer_FirstSolidHitTrace(lua_State * tolua_S)
 		}
 		// Get the params:
 		cWorld * world;
-		Vector3d * start;
-		Vector3d * end;
+		Vector3d start;
+		Vector3d end;
 		if (!L.GetStackValues(idx, world, start, end))
```

Comment:
Above this is `L.IsParamUserType(idx + 1, "Vector3<double>")` and `L.CheckParamUserType(idx + 1, "Vector3<double>", idx + 2)`. I think you need a `CheckParamVector3`.

Reply:
What an unbelievable catch

Reply:
You didn't address this fully. You didn't add a `CheckParamVector3` and if you search for `Vector3<double>` then you can see there is also an issue with `tolua_cLineBlockTracer_LineOfSightTrace`.